app/src/sharedTest/java/org/oppia/android/app/options/OptionsFragmentTest.kt

```diff
@@ -374,7 +387,7 @@ class OptionsFragmentTest {
       intended(
         allOf(
           hasExtra(
-            ReadingTextSizeActivity.KEY_READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE,
+            KEY_READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE,
```

Comment:
Let's use the value directly.
```suggestion
            ReadingTextSizeActivity.getKeyReadingTextSizePreferenceSummaryValue(),
```

Reply:
ditto for all